pkg/v1/builder/command/doc.go

```diff
@@ -1,5 +0,0 @@
-// Copyright 2021 VMware, Inc. All Rights Reserved.
```

Comment:
Could you keep the `doc.go` file? It makes sure that documentation for the package is not empty in https://pkg.go.dev/github.com/vmware-tanzu/tanzu-framework@v0.19.0/pkg/v1/builder/command

Reply:
I think I'd prefer `doc.go` files since we're already doing it in a bunch of places. I'd like to push for it as a convention.